Automattic/wp-desktop
# Framework: updated makefile to work with single build.js

Along with the `wp-calypso` [PR here](https://github.com/Automattic/wp-calypso/pull/11785) (which has already landed), this PR seeks make `wp-desktop` compatible with the latest changes to wp-calypso (single build for non-development envs).

### Testing
1. Checkout ~~`try/framework/ssr-config-desktop-compat`~~ `master` Calypso branch.
2. Run `make test` to verify it works properly.
3. Run `make run` and check if it builds and works properly.
